File: 344.反转字符串.py
#
# @lc app=leetcode.cn id=344 lang=python3
#
# [344] 反转字符串
#

# @lc code=start
class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        l, r = 0, len(s) - 1
        while l <= r:
            tem = s[l]
            s[l] = s[r]
            s[r] = tem
            l += 1
            r -= 1
        return s
# @lc code=end

File: 344/344.反转字符串.go
/*
 * @lc app=leetcode.cn id=344 lang=golang
 *
 * [344] 反转字符串
 */

// @lc code=start
func reverseString(s []byte)  {
	l, r := 0, len(s) - 1
	for l <= r {
		temp := s[l]
		s[l] = s[r]
		s[r] = temp
		l++
		r--
	}
}
// @lc code=end

